src/qt/bitcoin.cpp

```diff
@@ -302,7 +302,7 @@ void BitcoinCore::shutdown()
         threadGroup.join_all();
         Shutdown();
         qDebug() << __func__ << ": Shutdown finished";
-        Q_EMIT shutdownResult(1);
```

Comment:
The int return value was only use over qDebug which we can argue what it should have been there.
But this PRs solution (remove the shutdown result value) seems to make most sense.